code/modules/paperwork/paper.dm

```diff
@@ -88,6 +88,7 @@
 	return
 
 /obj/item/weapon/paper/attack_ai(var/mob/living/silicon/ai/user as mob)
+	user.delayNextAttack(10)
```

Comment:
No, it won't work, you're correct. The result of speed coding. I'll fix it tomorrow, pinkie swear.